### PR TITLE
RAC-5103 - Converted misc CIT formatted apiv2 scripts to FIT format - catalogs, etc.

### PR DESCRIPTION
**Pull request by hohene**
Converted the workflows CIT test scripts to FIT format.
This is first part of committing these changed scripts to show the diffs.
catalogs, configs, lookups, pollers

### CODE REVIEW

**Comment by hohene** [comment]
@larry-dean @keedya @jimturnquist @johren @derrickostertag

File: test/tests/api/fit-staging/v2_0/catalogs_tests.py [hunk 1]
```diff
@@ -1,67 +1,70 @@
...
 from on_http_api2_0 import rest
+from json import loads
+from nosedep import depends
+from nose.plugins.attrib import attr
 
+logs = flogging.get_loggers()
 
-LOG = Log(__name__)
 
+# @test(groups=['catalogs_api2.tests'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/catalogs_tests.py]: This should be removed (per Jeanne)

File: test/tests/api/fit-staging/v2_0/catalogs_tests.py [hunk 2]
```diff
@@ -1,67 +1,70 @@
...
                 for source in self.__expected_sources:
                     for catalog in catalogs:
                         if catalog.get('source') == source and catalog.get('node').find(node.get('id')):
                             break
                     else:
-                        fail('Catalog {0} not found in node {1}!'.format(source,node.get('id')))
+                        self.fail('Catalog {0} not found in node {1}!'.format(source, node.get('id')))
 
-    @test(groups=['catalogs_api2.tests'], depends_on_groups=['api2_check-catalogs'])
+    # @test(groups=['catalogs_api2.tests'], depends_on_groups=['api2_check-catalogs'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/catalogs_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/config_tests.py [hunk 3]
```diff
@@ -1,44 +1,47 @@
...
+        # """Testing GET:api/2.0/config to get server configuration"""
         Api().config_get()
         rsp = self.__client.last_response
-        assert_equal(200, rsp.status, message=rsp.reason)
+        self.assertEqual(200, rsp.status, msg=rsp.reason)
 
-    @test(groups=['api2_patch-config'], depends_on_groups=['api2_check-config'])
-    def patch_server_config(self):
-        """Testing PATCH:api/2.0/config to patch a specific configuration item"""
+    # @test(groups=['api2_patch-config'], depends_on_groups=['api2_check-config'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/config_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/config_tests.py [hunk 4]
```diff
@@ -1,44 +1,47 @@
...
+from nose.plugins.attrib import attr
 
-LOG = Log(__name__)
+logs = flogging.get_loggers()
 
 
-@test(groups=['config_api2.tests'])
-class ConfigTests(object):
-    def __init__(self):
+# @test(groups=['config_api2.tests'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/config_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/lookups_tests.py [hunk 5]
```diff
@@ -1,153 +1,169 @@
...
-    def check_lookups(self):
-        """ Testing GET:/lookups """
+from json import loads, dumps
+from nosedep import depends
+from nose.plugins.attrib import attr
+
+logs = flogging.get_loggers()
+
+
+# @test(groups=['lookups_api2.tests'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/lookups_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/lookups_tests.py [hunk 6]
```diff
@@ -1,153 +1,169 @@
...
             for id in entry:
                 Api().lookups_get_by_id(id.get('id'))
                 rsp = self.__client.last_response
-                assert_equal(200, rsp.status, message=rsp.reason)
+                self.assertEqual(200, rsp.status, msg=rsp.reason)
 
-    @test(groups=['api2_check-post-lookup'],depends_on_groups=['api2_check-lookups-query'])
-    def post_lookup(self):
-        """ Testing POST /"""
+    # @test(groups=['api2_check-post-lookup'],depends_on_groups=['api2_check-lookups-query'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/lookups_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/lookups_tests.py [hunk 7]
```diff
@@ -1,153 +1,169 @@
...
-
-    @test(groups=['api2_check-post-lookup-negativeTesting'],depends_on_groups=['api2_check-lookups-query','api2_check-post-lookup'])
-    def post_lookup_negativeTesting(self):
-        """ Negative Testing POST / """
-        #Validate that a POST for a lookup with same id as an existing one gets rejected
+        self.assertEqual(str(rsp[0].get("ipAddress")), str(self.lookup.get("ipAddress")))
+        self.assertEqual(str(rsp[0].get("macAddress")), str(self.lookup.get("macAddress")))
+        self.assertEqual(str(rsp[0].get("node")), str(self.lookup.get("node")))
+
+    # @test(groups=['api2_check-post-lookup-negativeTesting'],depends_on_groups=['api2_check-lookups-query','api2_check-post-lookup'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/lookups_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/lookups_tests.py [hunk 8]
```diff
@@ -1,153 +1,169 @@
...
+            self.assertEqual(400, e.status, msg='Expected 400 status, received {}'.format(e.status))
         except (TypeError, ValueError) as e:
-           assert(e.message)
-
-    @test(groups=['api2_check-patch-lookup'],depends_on_groups=['api2_check-lookups-query','api2_check-post-lookup','api2_check-post-lookup-negativeTesting'])
-    def patch_lookup(self):
-        """ Testing PATCH /:id"""
+            assert(e.message)
+
+    # @test(groups=['api2_check-patch-lookup'],depends_on_groups=['api2_check-lookups-query',
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/lookups_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/lookups_tests.py [hunk 9]
```diff
@@ -1,153 +1,169 @@
...
         rsp = loads(self.__client.last_response.data)
-        assert_equal(str(rsp[0].get("node")),self.patchedNode.get("node") )
-
-    @test(groups=['check-delete-lookup'], depends_on_groups=['api2_check-lookups-query','api2_check-post-lookup','api2_check-patch-lookup'])
-    def delete_lookup(self):
-        """ Testing DELETE /:id """
-        #Validate that the lookup is there before it is deleted
+        self.assertEqual(str(rsp[0].get("node")), self.patchedNode.get("node"))
+
+    # @test(groups=['check-delete-lookup'], depends_on_groups=['api2_check-lookups-query',
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/lookups_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/pollers_tests.py [hunk 10]
```diff
@@ -1,89 +1,105 @@
...
+class PollersTests(fit_common.unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.__client = config.api_client
+        cls.pollers = []
+        cls.poller_libs = []
+        cls.created_pollers = []
+
+    # @test(groups=['pollers_api2.tests', 'api2_get_pollers_library'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/pollers_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/pollers_tests.py [hunk 11]
```diff
@@ -1,89 +1,105 @@
...
 
-        self._poller_libs = data
+        self.__class__.poller_libs = data
 
-    @test(groups=['pollers_api2.tests', 'api2_get_pollers_library_by_id'],
-          depends_on_groups=['api2_get_pollers_library'])
-    def get_pollers_lib_by_id(self):
-        """Test GET:api/2.0/pollers/library/:identifier"""
-        for lib in self._poller_libs:
+    # @test(groups=['pollers_api2.tests', 'api2_get_pollers_library_by_id'],
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/pollers_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/pollers_tests.py [hunk 12]
```diff
@@ -1,89 +1,105 @@
...
         try:
             Api().pollers_lib_by_id_get(identifier='not_a_real_poller')
         except ApiException as e:
-            assert_true(404, e.status)
+            self.assertEqual(404, e.status, msg='Expected 404 status, received {}'.format(e.status))
 
-    @test(groups=['pollers_api2.tests', 'api2_get_pollers'])
-    def get_pollers(self):
-        """Test GET:api/2.0/pollers"""
+    # @test(groups=['pollers_api2.tests', 'api2_get_pollers'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/pollers_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/pollers_tests.py [hunk 13]
```diff
@@ -1,89 +1,105 @@
...
-    def get_pollers_by_id(self):
-        """Test GET:api/2.0/pollers/:identifier"""
-        for poller in self._pollers:
+            self.assertTrue('config' in keys)
+            self.assertTrue('type' in keys)
+            self.assertTrue('pollInterval' in keys)
+
+        self.__class__.pollers = data
+
+    # @test(groups=['pollers_api2.tests', 'api2_get_pollers_by_id'],
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/pollers_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/pollers_tests.py [hunk 14]
```diff
@@ -1,89 +1,105 @@
...
         try:
             Api().pollers_id_get(identifier='does_not_exist')
         except ApiException as e:
-            assert_true(404, e.status)
+            self.assertEqual(404, e.status, msg='Expected 404 status, received {}'.format(e.status))
 
-    @test(groups=['pollers_api2.tests', 'api2_create_pollers'])
-    def create_pollers(self):
-        """Test POST:api/2.0/pollers"""
+    # @test(groups=['pollers_api2.tests', 'api2_create_pollers'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/pollers_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/pollers_tests.py [hunk 15]
```diff
@@ -108,80 +124,83 @@ def create_pollers(self):
...
+                self.assertEqual(poller[key], data[key])
 
-            self._created_pollers.append(data)
+            self.__class__.created_pollers.append(data)
 
-    @test(groups=['pollers_api2.tests', 'api2_patch_pollers'],
-          depends_on_groups=['api2_create_pollers'])
-    def pollers_patch(self):
-        """Test PATCH:api/2.0/pollers/:identifier"""
+    # @test(groups=['pollers_api2.tests', 'api2_patch_pollers'],
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/pollers_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/pollers_tests.py [hunk 16]
```diff
@@ -108,80 +124,83 @@ def create_pollers(self):
...
-            assert_true(404, e.status)
-
-    @test(groups=['pollers_api2.tests', 'api2_data_get_pollers'],
-          depends_on_groups=['api2_get_pollers', 'api2_patch_pollers'])
-    def pollers_data_get(self):
-        """Test GET:/api/2.0/pollers/:identifier/data"""
-        all_pollers = self._pollers + self._created_pollers
+            self.assertEqual(404, e.status, msg='Expected 404 status, received {}'.format(e.status))
+
+    # @test(groups=['pollers_api2.tests', 'api2_data_get_pollers'],
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/pollers_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/pollers_tests.py [hunk 17]
```diff
@@ -108,80 +124,83 @@ def create_pollers(self):
...
-                assert_equal(len(result.data), 0, 'Poller data should have 0 length')
-
-    @test(groups=['pollers_api2.tests', 'api2_current_data_get_pollers'],
-          depends_on_groups=['api2_get_pollers', 'api2_data_get_pollers'])
-    def pollers_current_data_get(self):
-        """Test GET:/api/2.0/pollers/:identifier/data/current"""
-        all_pollers = self._pollers + self._created_pollers
+                self.assertEqual(len(result.data), 0, 'Poller data should have 0 length')
+
+    # @test(groups=['pollers_api2.tests', 'api2_current_data_get_pollers'],
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/pollers_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/pollers_tests.py [hunk 18]
```diff
@@ -108,80 +124,83 @@ def create_pollers(self):
...
-                assert_equal(len(result.data), 0, 'Poller data should have 0 length')
-
-    @test(groups=['pollers_api2.tests', 'api2_delete_pollers'],
-          depends_on_groups=['api2_current_data_get_pollers'])
-    def delete_pollers(self):
-        """Test DELETE:api/2.0/pollers/:identifier"""
-        for poller in self._created_pollers:
+                self.assertEqual(len(result.data), 0, 'Poller data should have 0 length')
+
+    # @test(groups=['pollers_api2.tests', 'api2_delete_pollers'],
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/pollers_tests.py]: remove

**Pull request merged**